pkg/interfaces/contracts/vault/IHooks.sol

```diff
@@ -136,15 +138,15 @@ interface IHooks {
     ***************************************************************************/
 
     /**
-     * @notice Optional hook to be executed before removing liquidity.
+     * @notice Hook to be executed before removing liquidity.
```

Comment:
Since we removed the Optional word, maybe we should explain that the hook is only executed if the flag is true? Not sure if this explanation is necessary in the code, just a suggestion.